connection.js

```diff
@@ -173,6 +173,11 @@ class Connection {
                 return;
             }
 
+            // Don't connect if it is not our turn.
+            if (gamedata.player_to_move !== this.bot_id) {
```

Comment:
Why revert this change we made recently?

Reply:
its a rebasing last minute mistake
thankfully you spotted it

fixed now :+1: